code/game/gamemodes/miniantags/slaughter/bloodcrawl.dm

```diff
@@ -68,13 +68,16 @@
 			for(var/i in 1 to 3)
 				playsound(get_turf(src), sound, 100, 1)
 				sleep(30)
-			if(kidnapped)
-				to_chat(src, "<B>You devour [kidnapped]. Your health is fully restored.</B>")
-				adjustBruteLoss(-1000)
-				adjustFireLoss(-1000)
-				adjustOxyLoss(-1000)
-				adjustToxLoss(-1000)
 
+			if(kidnapped)
+				if(ishuman(kidnapped) || isrobot(kidnapped))
+					to_chat(src, "<B>You devour [kidnapped]. Your health is fully restored.</B>")
+					adjustBruteLoss(-1000)
```

Comment:
isn't there like a 600 brute cap on brute damage?

Reply:
roughly, for humans yes. however a simple mob (such as a demon) has no cap on how much max health it could have.